cmd/buildah/pull.go

```diff
@@ -101,6 +105,17 @@ func pullCmd(c *cobra.Command, args []string, iopts pullOptions) error {
 		return err
 	}
 
+	decConfig := &encconfig.DecryptConfig{}
+	if len(iopts.decryptionKeys) > 0 {
+		// decryption
+		dcc, err := enchelpers.CreateCryptoConfig([]string{}, iopts.decryptionKeys)
+		if err != nil {
+			return errors.Wrapf(err, "Invalid decryption keys")
```

Comment:
Lowercase description.  Should this be made into a shared function, since we repeated it many times?

Reply:
Yea - agreed. let me refactor this.